### PR TITLE
snowflake connector

### PR DESCRIPTION
**Pull request by aswani521**
help doc for adding snowflake connector

### CODE REVIEW

File: pages/docs/tracking/integrations/snowflake.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+ℹ️ For a full list of event properties, please refer to [Events & Properties](https://docs.mixpanel.com/docs/tracking/reference/default-properties#event-properties)
+
+**The following columns are required:** 
+
+- Event Time
+- Distinct ID
+- Insert ID
+    - This is an optional field, but we recommend it as a best practice.
+- Insert Time
+    - We expect this to be a monotonically increasing number. We use this as a water mark to figure out which events are new and ingest them.
```
Comment (jakewski) [inline pages/docs/tracking/integrations/snowflake.md]: we expect this to be a `TIMESTAMP_NTZ` for snowflake

File: pages/docs/tracking/integrations/snowflake.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+## **Ingest Events**
+
+---
+
+Once you have created a warehouse source, follow the below steps to send events into Mixpanel.
+
+1. Go to Project Settings > Warehouse Data.
+2. Click on + Add Warehouse Event.
+3. Select the source from the dropdown (e.g. Snowflake, BigQuery).
```
Comment (jakewski) [inline pages/docs/tracking/integrations/snowflake.md:115]: maybe more like "Select the source you created earlier" - the user can have multiple sources to different snowflake instances

**Review by jakewski — APPROVED** [review]
LGTM good starting point, small nitpicks

File: pages/docs/tracking/integrations/snowflake.md (hunk 1 above)
Comment (aswani521) [reply pages/docs/tracking/integrations/snowflake.md]: fixed

**Pull request merged**